executor/analyze.go

```diff
@@ -320,8 +330,8 @@ func (e *AnalyzeIndexExec) buildStatsFromResult(result distsql.SelectResult, nee
 	return hist, cms, nil
 }
 
-func (e *AnalyzeIndexExec) buildStats() (hist *statistics.Histogram, cms *statistics.CMSketch, err error) {
-	if err = e.open(); err != nil {
+func (e *AnalyzeIndexExec) buildStats(ranges []*ranger.Range, considerNull bool) (hist *statistics.Histogram, cms *statistics.CMSketch, err error) {
```

Comment:
It seems that we don't update the `statistics.AnalyzeJob`'s `rowCount` in index analyze.

Reply:
Updated in https://github.com/pingcap/tidb/pull/10102/files#diff-a1052fd6e325ec9bbd1016c787b29196L307?